src/template.json

```diff
@@ -57,7 +57,7 @@
     "OwnerEmail": "khai.do@sagebase.org",
     "Project": "Infrastructure",
     "SnapshotGroups": "all",
-    "SourceImage": "ami-07ebfd5b3428b6f4d",
+    "SourceImage": "ami-0d0032af1da6905c7",
```

Comment:
Do you think we will never need a "base" AMI without docker?  For example, what if i want to create another AMI based off of a "sage" AMI that does not contain docker?  if you make this change then you've eliminated that possibility.  I would have to setup my own base AMI. 

Also if you go ahead with this then how about updating the README to indicate the contents and purpose of this AMI?

Reply:
I don't think there's a need for a ubuntu image that does not have the docker binaries on it. One of the patterns we want to enable for scientists is to push and pull from the Synapse container reg as a method of caching data, so I want to make that method available regardless of what kind of service catalog instance they pick. Note that having the binaries in the the base image does not mean dockerd will be running any containers until someone tell it to... We can also make it not start as a service on boot.

Reply:
ok